src/components/Scrollable/components/HorizontalScrollbar/HorizontalScrollbar.jsx

```diff
@@ -19,9 +19,11 @@ import Movable from 'components/Movable';
 import Context from '../../Scrollable.context';
 import {move} from './HorizontalScrollbar.operations';
 import './HorizontalScrollbar.scss';
+import {propTypes} from '../VerticalScrollbar/VerticalScrollbar.props';
 
-const HorizontalScrollbar = () => {
-    const track = useRef();
+const HorizontalScrollbar = ({xRef}) => {
```

Comment:
yes.. I know `xRef` vs `forwardRef` :) I just want as few functions in between when updating things on scroll events